# Terrain and request state pair definition
Header header

AtomicProposition[]  terrain_state
AtomicProposition[]  request_state  
